src/test/java/com/box/sdk/BoxEventTest.java

```diff
@@ -48,6 +48,9 @@ public void testIsEventLogUnique() throws IOException, ParseException {
         Date endDate = new SimpleDateFormat("yyyy-MM-dd'T'hh:mm:ssZ").parse(endTime);
         EventLog eventLog = EventLog.getEnterpriseEvents(this.api, startDate, endDate);
         Assert.assertTrue(eventLog.getSize() == 1);
+        for (BoxEvent event : eventLog) {
+            Assert.assertNotNull(event.getActionBy());
```

Comment:
Can we make this test a bit more strict?  Just saying that this isn't null doesn't give much confidence that it's actually working correctly